build_tools/packaging/linux/build_package.py

```diff
@@ -847,6 +795,12 @@ def clean_package_build_dir(artifacts_dir):
 
 
 def run(args: argparse.Namespace):
+    # Set the global variables
+    dest_dir = Path(args.dest_dir).expanduser().resolve()
+    global DEBIAN_CONTENTS_DIR, RPM_CONTENTS_DIR
```

Comment:
Do we really needs to be global? Can we define this as local variable and pass it as parameter in fn?